data/crd/ssp.kubevirt.io_ssps.yaml

```diff
@@ -14,7 +14,11 @@ spec:
     singular: ssp
   scope: Namespaced
   versions:
-  - name: v1beta1
+  - additionalPrinterColumns:
+    - jsonPath: .status.phase
+      name: Phase
+      type: string
+    name: v1beta1
```

Comment:
https://github.com/kubevirt/ssp-operator/pull/328 wasn't cherry picked back to 0.14 so should we drop this?

Reply:
Gah sorry it was included in [v0.14](https://github.com/kubevirt/ssp-operator/releases/tag/v0.14.0), ignore me. This looks fine.

/lgtm